examples/vkui-nextjs-ts/package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "vkui-nextjs-ts",
+  "version": "1.0.0",
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@vkontakte/icons": "latest",
+    "@vkontakte/vkui": "latest",
+    "next": "^15.0.0",
+    "react": "^19.0.0",
+    "react-dom": "^19.0.0"
+  },
+  "devDependencies": {
+    "@types/node": "latest",
+    "@types/react": "^19.0.0",
+    "@types/react-dom": "^19.0.0",
+    "eslint": "^8.56.0",
+    "eslint-config-next": "^15.0.0",
+    "typescript": "latest"
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

```suggestion
  }
```

PS: вообще должен использовать Yarn v4 как в корневом `package.json`

у тебя использует Yarn v1 скорей всего из-за того, что глобавльно 1-ый Yarn